lib/vintage_net_wizard.ex

```diff
@@ -22,7 +22,9 @@ defmodule VintageNetWizard do
   """
   @spec run_wizard([Endpoint.opt()]) :: :ok | {:error, String.t()}
   def run_wizard(opts \\ []) do
-    with :ok <- APMode.into_ap_mode(),
+    ifname = Keyword.get(opts, :ifname, "wlan0")
```

Comment:
```suggestion
    ifname = opts[:ifname] || "wlan0"
```

What do you think about a slightly stronger check here since this is a pretty crucial requirement? I'm starting to hesitate using the `.get/3` defaults because it doesn't account if the key is there, but is nil
```elixir
iex()> Keyword.get([ifname: nil], :ifname, "wat")
nil
```

But `nil` is not considered "truthy", so fetching and having the value == nil will default to the `||` clause

Reply:
Not that this makes sense in this context, but wouldn't you expect that if you set an option to `nil` that `nil` would be used. I.e., make it work like every other value rather than making `nil` sometimes mean `nil` and sometimes mean "use the default" depending on the API.

I was of the opinion that `ifname = opts[:ifname] || "wlan0"` was an anti-pattern, since it made `false` and `nil` special and would require the user to look at the docs or code to see what would happen.

Reply:
Thats a good point. I do think that allowing `nil` here would produce the same result of a breakage, confusion, and having to look at the code to deduce what is going on.

Maybe the better change is `raise "ifname cannot be nil"` (or returning `{:error, ...}` tuple)

Reply:
Rather than check for `nil`, I'd propose verifying that it's a string. I'd put that in a different PR, though, since it's more of a better error message kind of change.